.rubocop.yml

```diff
@@ -149,7 +149,7 @@ Style/IfUnlessModifier:
   Description: Favor modifier if/unless usage when you have a single-line body.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier
   Enabled: true
-  MaxLineLength: 100
+  MaxLineLength: 80
```

Comment:
I think we have a good number of files using 100 and github def. supports > 80. I'm fine with whatever we pick as long as we stick to it, with a slight preference towards 100.